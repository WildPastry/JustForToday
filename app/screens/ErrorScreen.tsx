import { FontDisplay } from '../components/styles/StyledText';
import { StyleSheet } from 'react-native';
import { View } from '../components/styles/Themed';

const ErrorScreen: React.FC = (): JSX.Element => {
  return (
    <View style={styles.container}>
      <FontDisplay style={styles.title}>ErrorScreen</FontDisplay>
    </View>
  );
};

const styles = StyleSheet.create({
  container: {
    alignItems: 'center',
    flex: 1,
    justifyContent: 'center',
    padding: 15
  },
  title: {
    fontSize: 20
  }
});

export default ErrorScreen;
