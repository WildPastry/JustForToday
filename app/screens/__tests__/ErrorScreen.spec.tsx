import ErrorScreen from '../ErrorScreen';
import renderer from 'react-test-renderer';

describe('<ErrorScreen />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<ErrorScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
