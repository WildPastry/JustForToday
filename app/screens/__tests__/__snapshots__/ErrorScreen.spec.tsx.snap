// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorScreen /> renders correctly 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "#fff",
      },
      {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
      },
    ]
  }
>
  <Text
    style={
      [
        {
          "color": "#000",
        },
        [
          {
            "fontSize": 20,
          },
          {
            "fontFamily": "space-mono",
          },
        ],
      ]
    }
  >
    ErrorScreen
  </Text>
</View>
`;
